assets/src/util/outfront.tsx

```diff
@@ -6,7 +6,7 @@ import { getDatasetValue } from "Util/dataset";
  *
  * Use this for DUP-specific logic.
  */
-export const isDup = () => /^file:.*dup-app.*/.test(location.href);
+export const isDup = () => /.*dup-app.*/.test(location.href);
```

Comment:
This needed to be changed because the `location.href` does not start with `file` in BrowserStack, but instead `YOUR_USERNAME.browserstack.com`. This seems like a strange thing to pattern match on since that's liable to change in the future.

I don't think removing the file from the pattern match should be a concern, since we would never use the lowercase string `dup-app` anywhere else